lms/envs/common.py

```diff
@@ -2125,20 +2125,27 @@
 ECOMMERCE_API_TIMEOUT = 5
 
 # PROFILE IMAGE CONFIG
-# TODO: add these settings to aws.py as well
 # WARNING: Certain django storage backends do not support atomic
-# file overwrites (including the default, specified below) - instead
+# file overwrites (including the default, OverwriteStorage) - instead
 # there are separate calls to delete and then write a new file in the
 # storage backend.  This introduces the risk of a race condition
 # occurring when a user uploads a new profile image to replace an
 # earlier one (the file will temporarily be deleted).
-PROFILE_IMAGE_BACKEND = 'storages.backends.overwrite.OverwriteStorage'
-# PROFILE_IMAGE_DOMAIN points to the domain from which we serve image
-# files from.  When this is '/', it refers to the same domain as the
-# app server.  If serving from a different domain, specify that here
-# i.e. 'http://www.example-image-server.com/'
-PROFILE_IMAGE_DOMAIN = '/'
-PROFILE_IMAGE_URL_PATH = 'media/profile_images/'
+PROFILE_IMAGE_BACKEND_CONFIG_AWS = {
+    'storage_class': 'storages.backends.s3boto.S3BotoStorage',
+    'storage_kwargs': {
+        'bucket': 'jsaedxtest',
+        'custom_domain': 's3.amazonaws.com/jsaedxtest',
+    },
+}
+PROFILE_IMAGE_BACKEND_CONFIG_FS = {
+    'storage_class': 'storages.backends.overwrite.OverwriteStorage',
+    'storage_kwargs': {
+        'location': '/edx/var/edxapp/uploads/profile_images/',
+        'base_url': '/media/profile_images/',
+    },
+}
+PROFILE_IMAGE_BACKEND_CONFIG = PROFILE_IMAGE_BACKEND_CONFIG_AWS  # DO NOT MERGE LIKE THIS, JUST FOR LOCAL DEBUGGING.
```

Comment:
it's not intended that we include multiple configurations in the app code, this is just to show what a well-formed configuration will look like for each backend.